### PR TITLE
Feature front

### PR DESCRIPTION
**Pull request by Quentin01**
Remake of #2 
Merge #3 first

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var app = express.Router();
+var mongoose = require('mongoose');
+var gpu = (require('../api/v1/models/gpu'))();
+
+app.use(express.static(__dirname));
+
+app.get('/', (req, res) => {
+	gpu.find().then((gpus) => {
```
Comment (Quentin01) [inline app/index.js]: There are indentation problems in that file (we use 4 spaces to indent)

File: server.js [hunk 2]
```diff
@@ -19,13 +19,16 @@ server.use(express.static(__dirname));
...
+server.set('views', __dirname + '/app/views');
 
 var api = require('./api');
+var app = require('./app');
 
 server.get('/', function (req, res) {
     res.sendFile('./index.html');
 });
 
+server.use('/app', app);
```
Comment (Quentin01) [inline server.js]: Why do you want an `/app`? Or if you still want that, maybe redirect `/` to `/app`?

**Review by yoannpicquenot — CHANGES_REQUESTED** [review]
.floo files have to be removed

**Review by yoannpicquenot — DISMISSED** [review]

**Comment by Quentin01** [comment]
Is it ready to be review?

**Comment by Quentin01** [comment]
@mapy69003 @cchardeau Does someone can answer my question? Is it ready to review or not?

**Comment by yoannpicquenot** [comment]
@Quentin01 Sorry, not yet

**Comment by yoannpicquenot** [comment]
@Quentin01 ready to be reviewed

**Comment by Sumenia** [comment]
@Quentin01 @saveman71 , you were both be assigned to review this PR since May 25. 
For the sake of the project, we need to merge this to dev/master asap. We cannot go on working on one single branch as the PR will only get bigger.

**Pull request closed without merging**